### PR TITLE
fix: privateKey can't be parsed by JS

### PR DESCRIPTION
**Pull request by sxiaojian88**
fix: privateKey can't be parsed by JS and generate same privateKey every time
fix: MASKMOBILE-382

### CODE REVIEW

File: Maskbook/Persona/Persona.swift [hunk 1]
```diff
@@ -68,14 +69,28 @@ extension Persona {
...
+            _ = try data.pack("key_ops")
+            _ = try data.pack(key_ops)
+            _ = try data.pack("kty")
+            _ = try data.pack(privateKey.kty ?? "")
+            _ = try data.pack("ext")
+            _ = try data.pack(true)
+            _ = try data.pack("crv")
+            _ = try data.pack(privateKey.crv ?? "")
+            _ = try data.pack("d")
+            _ = try data.pack(privateKey.d ?? "")
```
Comment (jk234ert) [inline Maskbook/Persona/Persona.swift:90]: How about put these lines into a function of JWK struct?

Comment (sxiaojian88) [reply Maskbook/Persona/Persona.swift:90]: ok

File: Maskbook/Persona/JsonWebKey.swift [hunk 2]
```diff
@@ -45,3 +46,36 @@ extension JsonWebKey {
         }
     }
 }
+
+extension JsonWebKey {
+    var privateKeyPackedData: Data? {
+        var data = Data()
```
Comment (foxsin10) [inline Maskbook/Persona/JsonWebKey.swift:52]: Maybe make `guard let key_ops = key_ops else { return nil }` before `var data = Data()`?

**Review by foxsin10 — APPROVED** [review]

File: Maskbook/Persona/JsonWebKey.swift (hunk 2 above)
Comment (sxiaojian88) [reply Maskbook/Persona/JsonWebKey.swift]: ok

**Pull request merged**